projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -292,6 +292,10 @@ private function build_task( $task ) {
 		$built_task['badge_text']   = $this->load_value_from_callback( $task, 'badge_text_callback' );
 		$built_task['isLaunchTask'] = isset( $task['isLaunchTask'] ) ? $task['isLaunchTask'] : false;
 
+		$repetition_count                 = $this->load_repetition_count( $task );
+		$built_task['repetition_count']   = $repetition_count ? $repetition_count : null;
+		$built_task['minimum_repetition'] = isset( $task['minimum_repetition_to_complete'] ) ? $task['minimum_repetition_to_complete'] : null;
```

Comment:
As written, this means we won't set the value to `0` when that's the correct value.

Might it not be better to set `repetition_count` conditionally based on `target_repetitions`/`minimum_repetition`?
```suggestion
		if ( isset( $task['target_repetitions' ] ) {
			$built_task['target_repetitions'] = $task['target_repetitions'];

			// Note: this logic assumes that we return a number from load_repetition_count()
			$repetition_count               = $this->load_repetition_count( $task );
			$built_task['repetition_count'] = $repetition_count;
		} else {
			// Not sure if we need this
			$built_task['target_repetitions'] = null;
			$built_task['repetition_count']   = null;
		}
```